frontend/src/app/data/providers/data/data.service.ts

```diff
@@ -14,9 +14,8 @@ export abstract class DataService<T> {
     return '/api/' + this.api + '/' + id;
   }
 
-  get(id: string = '', params: { limit?: number, skip?: number, search?: string } = {}): Observable<T | T[]> {
+  get(id: string = '', params: { limit?: number, skip?: number, search?: string, isUser?: string} = {}): Observable<T | T[]> {
```

Comment:
right now in feature/backend/search branch @nazmicancalik uses search by taking "isUser" boolean query parameter. Based on that I add this.